packages/typink/src/providers/WalletSetupProvider.tsx

```
@@ -0,0 +1,122 @@
+
+      // reset connected account on wallet changing
+      setAccounts([]);
+      removeConnectedAccount();
+      // TODO how should we pick the first/default account after wallet connected/switched
+
+      unsub = injected.accounts.subscribe(setAccounts);
+
+      setSigner(injected.signer as any);
+    })();
+
+    return () => unsub && unsub();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)